src/features/faq/faq.tsx

```diff
@@ -0,0 +1,84 @@
+import { TextWithLink, Title } from '@/app/components';
+
+import './faq.scss';
+
+const data = [
+  {
+    question: 'Где можно задать вопрос?',
+    answer: [
+      {
+        id: 0,
+        text: 'Вопросы можно задать в Discord ',
+        title: 'чате',
+        link: 'https://discord.gg/2Ww3TCBvz4',
+      },
+    ],
+  },
+  {
+    question: 'Где происходит общение?',
+    answer: [
+      {
+        id: 1,
+        text: 'В Discord ',
+        title: 'чате',
+        link: 'https://discord.gg/2Ww3TCBvz4',
+      },
+    ],
+  },
+  {
+    question: 'Имеет ли значения город проживания? Можно ли пройти курс полностью онлайн?',
+    answer: 'Город проживания значения не имеет. Все этапы обучения можно пройти онлайн.',
+  },
+  {
```

Comment:
@ansivgit Do we have a conclusion about where the data should be stored, probably this shouldn't be inside the component file, but stored with other data in related place

Reply:
I think it will be better to store it separate